src/hotspot/share/include/jvm.h

```diff
@@ -30,6 +30,8 @@
 
 #include "jni.h"
 #include "jvm_md.h"
+#include "jvm_constants.h"
+#include "jvm_io.h"
```

Comment:
Note that the comment block starting at line 40 needs to be updated now you have refactored things.

Reply:
I fixed the comments.